RandomNumber.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+public class GetValue
+{
+    public string name { get; } = "Bat";
+}
+class RandomNumber
+{
+    private string ran;
+    public RandomNumber()
+    {
+        
+    }
+    static void MyMethod(string myString)
+            {
+                Random rnd = new Random();
+                int rndNum = rnd.Next(1, 100);
+                Console.WriteLine(rndNum + myString);
+            }
+    static void Main(string[] args)
```

Comment:
there should be only one Main method for the entire project. Make MyMethod public and call it in Program.Main